webpack.config.js

```diff
@@ -41,7 +41,7 @@ const getWebpackConfig = (config, version) => {
         'process.env.BUILD_ENV': JSON.stringify(buildEnv),
       }),
       new IgnorePlugin({
-        resourceRegExp: /\/iconv-loader$/,
+        resourceRegExp: /\/iconv-loader$|\.eot$|\.svg$|\.ttf$|\.woff$|\.gif$/,
```

Comment:
Is this needed?

Reply:
Yea because for widgets that use the react-slick package it generates a bunch of useless files in the dist folder after running npm run build. 
![image](https://github.com/visenze/visenze-experiences-web/assets/70419463/2b3d27bc-4d9e-4193-b4a7-11c21cd6b497)